### ISSUE TITLE
Migrate content to docs/quick-docs?

### ISSUE THREAD

**Issue by sanjayankur31** [developer-portal/website#112]
I'd filed this in the content repo, but this seems like a better place. (https://github.com/developer-portal/content/issues/349)

Hi there,

Given that docs is now up and running, should the dev portal just be merged into them?

https://docs.fedoraproject.org/en-US/docs/

That way there's one place for users to find documentation, and all of docs can be translated etc too. A lot of the material on here seems like it'd fit right in with quick-docs for example:

https://docs.fedoraproject.org/en-US/quick-docs/

**Comment by pvalena** [comment]
Hello, the discussion of whether we should merge with docs aside (for now) - how, and who would convert, and merge it? 

Note that these are not supposed to be docs, also not strictly related to Fedora content (but usage ON Fedora). We have also some extensions and tests (and the scriplet testing is WIP).

See https://fedoraproject.org/wiki/Websites/Developer#About_The_Project.

**Comment by sanjayankur31** [comment]
There are automated pipelines that convert pages from different source to the asciidoc that the new website infrastructure uses. A large number of wiki pages were auto-converted, for example. The docs mailing list would be the best place to discuss this particular question.

Fedora Docs is not just technical release notes or the installation guide now. The idea of the new system is that any Fedora related documentation can be put up there, all in one place so that users can find all the info they need in one place. So, you'll see that the docs site has various bits---from how to use Fedora to do something, to the release notes, to quick recipes, to SOPs for various SIGs/teams. Quick docs literally is: "This is a collection of short HOWTO and FAQ style documentation for Fedora users". 

So, I think it'll be much more useful if all the info here moves to docs for users to find in one place. Whether it gets absorbed into quick-docs or merits a different isolated repo can be discussed. It'll also mean that the effort that is being used to develop the portal here can be diverted to improving the docs projects, which will benefit all its consumers.

**Comment by hroncok** [comment]
This could at least solve the problem with deployment :)

**Comment by Jibec** [comment]
This would also allow localization team to translate the good content available here.

**Comment by sanjayankur31** [comment]
As the wiki page for the dev portal lists the -devel list as the channel of communication, I've also started a thread here:

https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/ERIUN5KRWCV2Z46OI7ST5KCSSK2VESVF/

**Comment by pvalena** [comment]
If it's up to me (no notes from the other/previous maintainers), feel free to migrate the content.
I'd be hower sad to see the current functionality to go away, mainly:
 - mentions of individual page's authors
 - `edit` button on the page, leading to directly editing the page
 - web editor
 - no `git` knowledge needed to submit changes

Unfortunately, I don't have any spare cycles to do anything like that (I don't have much for the maintenance either).

**Comment by hroncok** [comment]
A month later, the website still shows very outdated content. If there is no reasonable way to solve that, I recommend that we migrate to Fedora docs instead. Showing outdated information to Fedora users is bad :/

**Comment by hroncok** [comment]
What can I do to move this forward?

**Comment by pvalena** [comment]
You're right, I didn't get around to fixing the build, sadly. I have one new interested maintainer though, so the build frequency may get better.

If you want to help with fixing the builds ASAP, there is PR #104 .

WRT the migration, I wasn't fully convinced (I consider the conversation ongoing), but I won't be blocking it if it's the generally favored solution. However I do not have cycles required to realize it.

More info in the ML conversation: 
https://lists.fedoraproject.org/archives/list/developer-portal@lists.fedoraproject.org/thread/OYG2XESOXWWB56V57LWNMQQB5KJ3JVOZ/

**Comment by pvalena** [comment]
FYI new release is coming. Builds (releases) are now semi-automated, and use Fedora in container: 

https://lists.fedoraproject.org/archives/list/developer-portal@lists.fedoraproject.org/thread/NKYSVEDIZYFTJXINAU7WFNZLQE5XR33A/

**Comment by hroncok** [comment]
Awesome!

**Comment by pvalena** [comment]
Closing, as there're multiple unanswered questions WRT migration plan (and lack of someone who will take ownership thereof).
